pkgs/development/libraries/frozen/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, writeText
+}:
+
+stdenv.mkDerivation rec {
+  pname = "frozen";
+  version = "unstable-2021-02-23";
+
+  src = fetchFromGitHub {
+    owner = "cesanta";
+    repo = "frozen";
+    rev = "21f051e3abc2240d9a25b2add6629b38e963e102";
+    sha256 = "sha256-BpuYK9fbWSpeF8iPT8ImrV3CKKaA5RQ2W0ZQ03TciR0=";
+  };
+
+  buildInputs = [];
+  nativeBuildInputs = [ meson ninja ];
+
+  # frozen only has internal make and bazel support
+  # as it only consists of two source files, adding a
+  # small meson file on the fly is the easiest option for
+  # cross compilation support
```

Comment:
Would it be feasible to get this support upstream? Even if not, it'd probably better to open a PR and pull in the patch via the GitHub API from the PR.

Reply:
If the PR gets closed, the patch disappears though — learned this the hard way with ZFS :D.

Reply:
https://github.com/cesanta/frozen/pull/71